meta-ibm/recipes-kernel/linux/linux-aspeed/p10bmc.cfg

```diff
@@ -1,3 +1,7 @@
+CONFIG_AUDIT=y
```

Comment:
Are these required for bmcweb to work with the audit function enabled? If so, we should probably just put these in the meta-aspeed layer to ensure they are always there when enabled in bmcweb. Is there a significant size or performance impact to having these enabled by default for all systems?

Reply:
Yes, just checked and the logs are not produced if these are not present. Should a seperate p10bmc.cfg be created, or  should these be added to one of the cfg files already present in meta-aspeed

Reply:
I don't have the expertise to answer this one. If it doesn't hurt anything, and the logs are just not shown, maybe it's ok to just do in meta-ibm? You could try asking in discord what peoples thoughts are or just submit the change up to the defconfig files under https://grok.openbmc.org/xref/openbmc/openbmc/meta-aspeed/recipes-kernel/linux/linux-aspeed/ and see what the maintainers think.